db/data_migration/20160516075931_remove_draft_with_too_many_attachments.rb

```diff
@@ -0,0 +1 @@
+Whitehall.edition_services.deleter(DetailedGuide.find(621922).perform!
```

Comment:
Oof, amateur hour. Thanks, updated. Added a check for the record to make the migration less fragile.